### PR TITLE
Added ability to exclude selected tags from search

### PR DESCRIPTION
**Pull request by Acu1000**

### CODE REVIEW

File: Scarab/Services/Installer.cs [hunk 1]
```diff
@@ -12,7 +12,6 @@
 using Avalonia.Threading;
 using MsBox.Avalonia.Enums;
 using Microsoft.Toolkit.HighPerformance;
-using MsBox.Avalonia.Enums;
```
Comment (TheMulhima) [inline Scarab/Services/Installer.cs:15]: Could you undo this. I have local changes that would conflict with this

File: Scarab/Services/Interfaces/IUrlSchemeHandler.cs [hunk 2]
```diff
@@ -2,7 +2,6 @@
 using System.Threading.Tasks;
 using MsBox.Avalonia.Dto;
 using MsBox.Avalonia.Enums;
-using MsBox.Avalonia.Enums;
```
Comment (TheMulhima) [inline Scarab/Services/Interfaces/IUrlSchemeHandler.cs:5]: Could you undo this. I have local changes that would conflict with this

File: Scarab/Services/UrlSchemeHandler.cs [hunk 3]
```diff
@@ -12,7 +12,6 @@
 using MsBox.Avalonia.Dto;
 using Avalonia.Threading;
 using MsBox.Avalonia.Enums;
-using MsBox.Avalonia.Enums;
```
Comment (TheMulhima) [inline Scarab/Services/UrlSchemeHandler.cs:15]: Could you undo this. I have local changes that would conflict with this

File: Scarab/Util/DisplayErrors.cs [hunk 4]
```diff
@@ -9,7 +9,6 @@
 using Avalonia.Controls;
 using Avalonia.Controls.ApplicationLifetimes;
 using MsBox.Avalonia.Enums;
-using MsBox.Avalonia.Enums;
```
Comment (TheMulhima) [inline Scarab/Util/DisplayErrors.cs:12]: Could you undo this. I have local changes that would conflict with this

File: Scarab/Util/MessageBoxUtil.cs
Comment (TheMulhima) [inline Scarab/Util/MessageBoxUtil.cs:1]: Could you undo this. I have local changes that would conflict with this

File: Scarab/ViewModels/MainWindowViewModel.cs [hunk 5]
```diff
@@ -22,7 +22,6 @@
 using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
-using MsBox.Avalonia.Enums;
```
Comment (TheMulhima) [inline Scarab/ViewModels/MainWindowViewModel.cs:25]: Could you undo this. I have local changes that would conflict with this

**Review by TheMulhima — CHANGES_REQUESTED** [review]

File: Scarab/Services/Installer.cs (hunk 1 above)
Comment (Acu1000) [reply Scarab/Services/Installer.cs]: done

**Pull request merged**